lib/dispatch_web/webhooks/controller.ex

```diff
@@ -6,22 +6,26 @@ defmodule DispatchWeb.Webhooks.Controller do
   def create(conn, %{"action" => "opened", "pull_request" => %{"title" => "[WIP]" <> _}}), do: json(conn, %{success: true, noop: true})
 
   def create(conn, %{"action" => "opened", "pull_request" => %{"user" => %{"type" => "Bot"}}}), do: json(conn, %{success: true, noop: true})
+  def create(conn, %{"action" => "opened", "pull_request" => %{"draft" => true}}), do: json(conn, %{success: true, noop: true})
 
-  def create(conn, %{"action" => "opened", "repository" => %{"owner" => %{"login" => repo_owner}}} = params) do
+  def create(conn, %{"action" => "opened", "repository" => %{"owner" => %{"login" => repo_owner}}} = params), do: create_if_owner(conn, params, repo_owner)
+  def create(conn, %{"action" => "ready_for_review", "repository" => %{"owner" => %{"login" => repo_owner}}} = params), do: create_if_owner(conn, params, repo_owner)
+
+  def create(conn, _), do: json(conn, %{success: true, noop: true})
+
+  defp create_if_owner(conn, params, repo_owner) do
     github_organization_login = github_organization_login()
 
     case repo_owner do
       ^github_organization_login ->
-        do_create(conn, params)
+        assign_reviewers(conn, params)
 
       _ ->
         json(conn, %{success: true, noop: true})
     end
   end
 
-  def create(conn, _), do: json(conn, %{success: true, noop: true})
-
-  defp do_create(conn, params) do
+  defp assign_reviewers(conn, params) do
```

Comment:
Is there a reason why you renamed this function?

Reply:
I originally wanted to name `create_if_owner` `do_create` like we often do when we wish to re-use the result of arguments matching... I think I changed my mind and settled for a clearer name and forgot I had renamed the other one.

In any case, I can change it back, but I think the new name is slightly more explicit in what the function accomplishes.

Reply:
The `do_<foo>` pattern is used in Elixir when validations cannot be ran as guard clauses. What we could do here is keep everything the way it was and just add this

```diff
-def create(conn, %{"action" => "opened", "repository" => %{"owner" => %{"login" => repo_owner}}} = params) do
+def create(conn, %{"action" => action, "repository" => %{"owner" => %{"login" => repo_owner}}} = params) when action in ~w(opened ready_for_review) do
```

There you go!

Reply:
That's indeed quite simpler. Thanks!